### DISCUSSION TITLE
https://docs.github.com/authentication/securing-your-account-with-two-factor-authentication-2fa

### DISCUSSION THREAD

**Discussion by LaRingkd** [community/community#126853] [Apps, API and Webhooks] [answered]
### Select Topic Area

Show & Tell

### Body

https://support.github.com/contact?flow=log_request&tags=rr-log-req

**Comment by HaShingThang** [comment, marked as answer]
**To secure your GitHub account with two-factor authentication (2FA), follow these steps:**

1. **Navigate to Settings:**
Click on your profile picture in the top-right corner and select Settings.

2. **Access Security Settings:**
In the left sidebar, click on Security.

3. **Enable 2FA:**
Under the Two-factor authentication section, click on Enable two-factor authentication.

4. **Choose a 2FA Method:**

 - **Using an Authentication App:**

   - Select Set up using an app.
   - Scan the QR code using an authentication app like Google Authenticator or Authy.
   - Enter the code generated by the app to verify.

 - **Using SMS:**

   - Select Set up using SMS.
   - Enter your mobile phone number.
   - Enter the code sent to your phone to verify.

5. **Recovery Codes:**
Save your recovery codes in a secure location. These can be used to access your account if you lose access to your 2FA device.

For a detailed guide, refer to the [GitHub documentation on securing your account with 2FA](https://docs.github.com/authentication/securing-your-account-with-two-factor-authentication-2fa).

By following these steps, you can add an extra layer of security to your GitHub account, protecting it from unauthorized access.
